### DISCUSSION TITLE
[gatsby-plugin-image] image gets cropped

### DISCUSSION THREAD

**Discussion by JenniferFalkenstein** [gatsbyjs/gatsby#35099] [Help]
Hey there,
I'm currently working on migrating `gatsby-plugin-image` to my project. Before I used the old `gatsby-image`
Now I have the problem, that the new plugin crops the height of my image and doesn't query the whole height, like it did before:
Before:
<img width="383" alt="Bildschirmfoto 2022-03-10 um 12 26 39" src="https://user-images.githubusercontent.com/80680536/157653994-ed8fe8b1-2972-44a6-b4a7-2808b0febbed.png">
After:
<img width="381" alt="Bildschirmfoto 2022-03-10 um 12 27 50" src="https://user-images.githubusercontent.com/80680536/157654021-3d9caf58-a6af-47c7-9013-b20df2153a72.png">

I don't know, what configuration I did wrong, maybe you have an idea?
Here's my graphql-query:
```
query {
      frontMobile: file(relativePath: { eq: "image-mobile.png" }) {
        childImageSharp {
          gatsbyImageData(
            quality: 100,
            width: 390
          )
        }
      }
}
```
I also didn't apply any other styles on the `<GatsbyImage />`-element

**Comment by AntonZelenkov1997** [comment]
I have exactly the same problem. Maybe it's because the height of the image is much larger than the width?

**Reply by AntonZelenkov1997** [reply]
I understood why I have this issue. It's because I'm using `withArtDirection`. There is a good [article](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#withartdirection) about this.
